website/templates/project/registration_editor_templates.mako

```diff
@@ -154,4 +154,28 @@
     </div>
 </script>
 
+<script>
+ function setAllBoxes(value) {
+     var boxes = document.querySelectorAll('input[type="checkbox"]');
+     $.each(boxes, function(i, box) {
+         this.checked = value;
+     });
+ }
+</script>
+
+<script type="text/html" id="importContributors">
+    <div col-lg-12>
+        <p>Select: <a onClick="setAllBoxes(true)">All</a> | <a onClick="setAllBoxes(false)">None</a></p>
+    </div>
+    <div data-bind="foreach: {data: contributors, as: 'contrib'}">
+        <div class="checkbox">
+            <label>
+                <input type="checkbox" data-bind="value: contrib">
+                <!-- ko text: contrib --> <!-- /ko -->
```

Comment:
Please use a span instead of this syntax: `<span data-bind=text: contrib></span>`